research/picai/central/launch.slrm

```diff
@@ -1,15 +1,14 @@
 #!/bin/bash
-
-#SBATCH --nodes=1
-#SBATCH --ntasks=1
-#SBATCH --ntasks-per-node=1
-#SBATCH --cpus-per-task=8
-#SBATCH --error=%j_%x.err
-#SBATCH --qos normal
+#
 #SBATCH --gres=gpu:1
+#SBATCH --qos=normal
 #SBATCH --mem=64GB
+#SBATCH -c 16
+#SBATCH --time=12:00:00
+#SBATCH --signal=B:USR1@60
```

Comment:
For my own learning, I assume the components of `B:USR1@60` has special meaning. What do each of the parts mean?

Reply:
The signal option sends a job a specified signal when it is within sig_time seconds of its end time. Here B means that we want to send the signal only to the bash shell. By default all job steps will be signaled, but not the batch shell itself (ie sub jobs created within the batch shell with srun). USR1 is the signal we want to raise (aka SIGUSR1 which you can see we are trapping at the bottom the slurm script ). 60 is the amount of seconds before the job ending we send the signal.

Reply:
Cool thanks for the additional info!

Reply:
No problem, it was news to me!